### PR TITLE
[IGNORE] trying to make travis bus build fail, to get an image of failed travis

### PR DESCRIPTION
**Pull request by xingyif**

### CODE REVIEW

File: spec/models/affiliate_spec.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 RSpec.describe Affiliate, type: :model do
   describe 'associations' do
     it 'should have many coordinators' do
-      should have_many(:coordinators)
+      should have_many(:coordinator)
```
Comment (zack) [inline spec/models/affiliate_spec.rb:6]: looks like you've got a typo here

**Review by zack — CHANGES_REQUESTED** [review]
please fix your typo

**Comment by xingyif** [comment]
@zack did that intentionally to make travis fail, so that i can get a screenshoot

**Comment by zack** [comment]
image

**Comment by xingyif** [comment]
@zack I used the `request change` screenshot too. so it's not a complete *useless* joke :)

**Pull request closed without merging**